### DISCUSSION TITLE
Can YOURLS support automatic upgrades in the future?

### DISCUSSION THREAD

**Discussion by lidongyangphd** [YOURLS/YOURLS#3115] [Ideas]
I think it would be really helpful to some people who just want the peace of mind to set their YOURLS instance to automatically upgrade itself whenever a new update is available. The update YOURLS message in the Admin panel is helpful but I just wish that there is an option for lazy people like me.

Thanks

**Comment by ozh** [comment]
Implementation of an updater within YOURLS is unlikely. That could be achieved with a plugin for sure.

This is also easily feasible with an external script that would check for a new version, download the zip, extract and run the updater.

**Reply by lidongyangphd** [reply]
Thanks for the response! I think a plugin should be a good solution. And thank you for this amazing software that I have been using since 2017!
